java/runtime-common/src/main/java/org/ray/spi/LocalSchedulerLink.java

```diff
@@ -17,17 +17,9 @@
 
   void reconstructObject(UniqueID objectId, boolean fetchOnly);
 
+  void reconstructObjects(byte[][] objectIds, boolean fetchOnly);
```

Comment:
we should make the parameter type of these 2 method consistent, use `List<UniqueID>`.